zk.js/src/utxo/select-in-utxos.ts

```diff
@@ -132,9 +137,9 @@ export function selectInUtxos({
   publicAmountSpl?: BN;
   publicAmountSol?: BN;
   rpcFee?: BN;
-  utxos?: Utxo[];
-  inUtxos?: Utxo[];
-  outUtxos?: OutUtxo[];
+  utxos?: (Utxo | ProgramUtxo<PlaceHolderTData>)[];
+  inUtxos?: (Utxo | ProgramUtxo<PlaceHolderTData>)[];
+  outUtxos?: (OutUtxo | ProgramOutUtxo<PlaceHolderTData>)[];
```

Comment:
I am not sure this function makes sense for program utxos.

Reply:
good point. this is quite intertwined with gettxparams/user class; will address when removing user class/taking care of the private txs side